conda_build/utils.py

```diff
@@ -1406,6 +1406,7 @@ class LoggingContext:
         "urllib3.connectionpool",
         "conda_index",
         "conda_index.index",
+        "conda_index.index.convert_cache",
```

Comment:
If an existing old-style "many small files" cache exists, it's converted to sqlite. Prints logs, fails tests that want no logs, unless we add it here.